opm/autodiff/BlackoilTransportModel.hpp

```diff
@@ -158,6 +158,13 @@ namespace Opm {
 
 
 
+        std::vector<V>
+        computeFluidInPlace(const ReservoirState& x,
+                            const WellState& xw,
+                            const std::vector<int>& fipnum)
+        {
+            return asImpl().computeFluidInPlace(x, xw, fipnum);
+        }
```

Comment:
> flow_sequential complains segmentation fault here

We should try to figure out why. Did you use a debugger to see what causes the segfault?